### PR TITLE
Laboratory work #2, Alexandr Ershov - 23FPL2 

### PR DESCRIPTION
**Pull request by LexanDar11o6**

### CODE REVIEW

**Comment by EIzharikov** [comment]
Unfortunately, there is nothing to watch. I advise you to hurry up and wish you good luck!

**Comment by EIzharikov** [comment]
> Unfortunately, there is nothing to watch. I advise you to hurry up and wish you good luck!

the same...

File: lab_2_retrieval_w_bm25/main.py [hunk 1]
```diff
@@ -95,13 +125,13 @@ def calculate_tf_idf(tf: dict[str, float], idf: dict[str, float]) -> dict[str, f
...
 
 def calculate_bm25(
-    vocab: list[str],
-    document: list[str],
-    idf_document: dict[str, float],
-    k1: float = 1.5,
-    b: float = 0.75,
-    avg_doc_len: float | None = None,
-    doc_len: int | None = None,
+        vocab: list[str],
```
Comment (EIzharikov) [inline lab_2_retrieval_w_bm25/main.py:128]: revert

File: lab_2_retrieval_w_bm25/main.py [hunk 2]
```diff
@@ -123,7 +153,7 @@ def calculate_bm25(
 
 
 def rank_documents(
-    indexes: list[dict[str, float]], query: str, stopwords: list[str]
+        indexes: list[dict[str, float]], query: str, stopwords: list[str]
```
Comment (EIzharikov) [inline lab_2_retrieval_w_bm25/main.py:156]: revert

File: lab_2_retrieval_w_bm25/main.py [hunk 3]
```diff
@@ -141,14 +171,14 @@ def rank_documents(
...
-    document: list[str],
-    idf_document: dict[str, float],
-    alpha: float,
-    k1: float = 1.5,
-    b: float = 0.75,
-    avg_doc_len: float | None = None,
-    doc_len: int | None = None,
+        vocab: list[str],
+        document: list[str],
+        idf_document: dict[str, float],
```
Comment (EIzharikov) [inline lab_2_retrieval_w_bm25/main.py:176]: revert

File: lab_2_retrieval_w_bm25/start.py [hunk 4]
```diff
@@ -26,7 +27,9 @@ def main() -> None:
             documents.append(file.read())
     with open("assets/stopwords.txt", "r", encoding="utf-8") as file:
         stopwords = file.read().split("\n")
-    result = None
+    # print(tokenize(documents[0]))
```
Comment (EIzharikov) [inline lab_2_retrieval_w_bm25/start.py:30]: rm comments

**Review by EIzharikov — COMMENTED** [review]
It seems strange that you done this lab by 4 with only one commit. Make CI green, answer&fix all my comments and be ready for Friday. Good luck.

File: lab_2_retrieval_w_bm25/main.py (hunk 1 above)
Comment (LexanDar11o6) [reply lab_2_retrieval_w_bm25/main.py:139]: didn't touch it

File: lab_2_retrieval_w_bm25/main.py (hunk 3 above)
Comment (LexanDar11o6) [reply lab_2_retrieval_w_bm25/main.py:187]: I don't see this code in pycharm, I don't know what it is

File: lab_2_retrieval_w_bm25/start.py (hunk 4 above)
Comment (LexanDar11o6) [reply lab_2_retrieval_w_bm25/start.py]: done

**Pull request closed without merging**